presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestDistributedQueries.java

```diff
@@ -132,6 +132,18 @@ public void testCreateTable()
     public void testCreateTableAsSelect()
             throws Exception
     {
+        assertQueryTrue("CREATE TABLE test_create_table_as_select_if_not_exists (a bigint, b double)");
```

Comment:
This is a bit long, how about removing "select" from the table name: `test_create_table_as_if_not_exists`